src/test/java/hudson/plugins/git/GitStatusMultipleSCMTest.java

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
Sorry I missed this in my first review. Could you remove this comment generated from your IDE?